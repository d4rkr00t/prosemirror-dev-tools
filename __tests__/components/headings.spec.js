import React from "react";
import renderer from "react-test-renderer";
import {
  Heading,
  HeadingWithButton,
  HeadingButton
} from "../../src/components/heading";

test("Heading renders without crashing", () => {
  var heading = renderer.create(<Heading />).toJSON();
  expect(heading).toMatchSnapshot();
});

test("HeadingWithButton renders without crashing", () => {
  var headingWithButton = renderer.create(<HeadingWithButton />).toJSON();
  expect(headingWithButton).toMatchSnapshot();
});

test("HeadingButton renders without crashing", () => {
  var headingButton = renderer.create(<HeadingButton />).toJSON();
  expect(headingButton).toMatchSnapshot();
});
