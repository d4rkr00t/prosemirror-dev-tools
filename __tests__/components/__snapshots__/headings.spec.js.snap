// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading renders without crashing 1`] = `
<h2
  className="sc-bdVaJa hnnrkQ"
/>
`;

exports[`HeadingButton renders without crashing 1`] = `
<button
  className="sc-htpNat hbwhGQ"
/>
`;

exports[`HeadingWithButton renders without crashing 1`] = `
<div
  className="sc-bwzfXH jIEWCi"
/>
`;
